app/api/attendees.py

```diff
@@ -31,6 +31,12 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
+        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
```

Comment:
Why any of these changes in this PR? Completely unrelated to the the issue and title.